libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -203,3 +201,45 @@
     text-decoration: underline;
   }
 }
+
+@media (min-width: 1200px) {
+  .feds-crossCloudMenu-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    border-top: solid 1px #f3f3f3;
+    background-color: #fff;
+  }
+
+  .feds-crossCloudMenu {
+    width: 100%;
+    max-width: var(--feds-maxWidth-nav);
+  }
+
+  .feds-crossCloudMenu ul {
+    padding: 20px 0;
+    display: flex;
+  }
+
+  .feds-crossCloudMenu-item:not(:last-child):after {
+    content: '|';
+  }
+
+  .feds-crossCloudMenu-item {
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  .feds-crossCloudMenu-item u {
```

Comment:
Would this be a common use-case? We usually cancel it for anchor elements, but I haven't seen any underline ones